docs/source/conf.py

```diff
@@ -386,7 +392,23 @@ def autodoc_skip_member(app, what, name, obj, skip, options):
             return True  # return True to exclude from docs.
 
 
+def linkcode_resolve(domain, info):
+    """Used for [source] links in documentation. After
+    https://github.com/sphinx-doc/sphinx/issues/1556 is closed, we can update the links to be more
+    specific."""
+    class_name = info['fullname'].split('.')[0]
+    module = info['module']
+    combined = f"{module}.{class_name}"
+    if class_name == module.split('.')[-1]:
+        combined = module  # Edge case where module is telegram.ext.ExtBot and class name is ExtBot
```

Comment:
? on v14, the module is `tg.ext._extbot`

Reply:
Yea I don't why it shows like that. It's not supposed to show the .ExtBot in the first place